haystack/nodes/prompt/prompt_node.py

```diff
@@ -365,15 +364,22 @@ def invoke(self, *args, **kwargs):
 
     @classmethod
     def supports(cls, model_name_or_path: str) -> bool:
-        if not all(m in model_name_or_path for m in ["google", "flan", "t5"]):
-            return False
-
         try:
-            # if it is google flan t5, load it, we'll use it anyway and also check if model loads correctly
-            AutoModelForSeq2SeqLM.from_pretrained(model_name_or_path)
-        except EnvironmentError:
+            config = AutoConfig.from_pretrained(model_name_or_path)
```

Comment:
Brilliant!

Reply:
Thanks! It's still not 100% ideal, since if a user were to load a private model this will fail. I don't think it is easy to address in this PR, so I'll open a new issue for it.